tools/generate_flag_testdata.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+templatize_help_text() {
+    $1 --help 2>&1 | \
+        sed 's/{{/{{ "{{/' | \
+        sed 's/}}/}}" }}/' | \
+        sed 's/Paths to search for config files in. (default .*)/Paths to search for config files in. (default [{{ .Workdir }}])/'
```

Comment:
This is slightly better (one invocation of sed):
```suggestion
        sed -e 's/{{/{{ "{{/' \
            -e 's/}}/}}" }}/' \
            -e  's/Paths to search for config files in. (default .*)/Paths to search for config files in. (default [{{ .Workdir }}])/'
```